src/app/shared/components/select-country/select-country.component.html

```diff
@@ -1,7 +1,6 @@
 <mat-form-field appearance="outline">
-  <mat-select placeholder="{{ 'SETTINGS.SELECT_COUNTRY' | translate}}">
-    <!-- <mat-option *ngFor="let country of countries | keyvalue">
-      <span class="flag-icon flag-icon-{{country.key | lowercase}}"></span> {{ country.value }}
-    </mat-option> -->
+  <mat-label>{{ "SETTINGS.COUNTRY" | translate }}</mat-label>
+  <mat-select placeholder="{{ 'SETTINGS.SELECT_COUNTRY' | translate}}" [(ngModel)]="selectedCountry">
+    <mat-option *ngFor="let country of countriesList" [value]="country.name">{{ country.name }}</mat-option>
```

Comment:
I think it's better to use [value]="country. iso3166CountryCode"

Reply:
Good idea! Done.